Sources/Quick/Configuration/Configuration.swift

```diff
@@ -103,10 +109,16 @@ final public class Configuration: NSObject {
         is provided with metadata on the example that the closure is being
         run after.
     */
+#if !os(Linux)
```

Comment:
Same as above.

I should mention, it's very cool that these conditional macros are available in Swift, and this is a great solution to the problem here--thanks, @briancroom!

> Aside: At her Brooklyn Swift talk @tamarnachmany spoke about giving constructive feedback to theater performers. The difference between "That was a stellar rehearsal. [...] Oh, one note, you messed up a line towards the end." and "You messed up a line at the end. Excellent rehearsal, though." is **huge**!
>
> In that respect, I now see why some people prefer code review tools like Phabricator or Gerrit--when you click "submit", an overall comment is displayed first, then in-line comments. On GitHub, you'll read my in-line nitpicks before you read my overarching comment on the pull request. That's a shame, because this is amazing work and it's much appreciated, even though my in-line comments focus on what could be improved.